eng/native/ijw/IJW.cmake

```diff
@@ -31,6 +31,19 @@ if (CLR_CMAKE_HOST_WIN32)
     set_target_properties(${targetName} PROPERTIES COMPILE_OPTIONS "${compileOptions}")
   endfunction()
 
+  function(add_ijw_msbuild_project_properties targetName ijwhost_target)
```

Comment:
Why is ijwhost_target needed here?

Reply:
I wanted a customization layer here as the name ijwhost isn't required to be the same (and the name for the fake implementation in src/tests has changed at times)